pdns/dnsdistdist/dnsdist-lua.cc

```diff
@@ -760,7 +762,7 @@ static void setupLuaConfig(LuaContext& luaCtx, bool client, bool configCheck)
                          }
 
                          dnsdist::configuration::updateRuntimeConfiguration([&server](dnsdist::configuration::RuntimeConfiguration& config) {
-                           config.d_backends.erase(std::remove(config.d_backends.begin(), config.d_backends.end(), server), config.d_backends.end());
+                           boost::range::remove_erase(config.d_backends, server);
```

Comment:
I would rather not introduce a `boost::range` for such a trivial change.

Reply:
Removed.